EpsilonAuxBMS/Src/SetContactorsTask.c

```diff
@@ -332,12 +343,19 @@ void setContactorsTask(void const* arg)
             }
 
             case BLOCKED:
-                common = !HAL_GPIO_ReadPin(COMMON_SENSE_GPIO_Port, COMMON_SENSE_Pin);
-                charge = !HAL_GPIO_ReadPin(CHARGE_SENSE_GPIO_Port, CHARGE_SENSE_Pin);
-                discharge = !HAL_GPIO_ReadPin(DISCHARGE_SENSE_GPIO_Port, DISCHARGE_SENSE_Pin);
+            {
+                uint8_t common = !HAL_GPIO_ReadPin(COMMON_SENSE_GPIO_Port, COMMON_SENSE_Pin);
+                uint8_t charge = !HAL_GPIO_ReadPin(CHARGE_SENSE_GPIO_Port, CHARGE_SENSE_Pin);
+                uint8_t discharge = !HAL_GPIO_ReadPin(DISCHARGE_SENSE_GPIO_Port, DISCHARGE_SENSE_Pin);
+
 
+                // If orion isn't overiding the contactors, but they are supposed to have been set, but aren't
+                // , this is an error condition and all contactors should be disconnected.
+                // Previously didn't check that orion was overiding contactors
                 if (prevState > COMMON_ENABLE_CHECK
-                        && (!common || (hasChargeBeenSet && !charge) || (hasDischargeBeenSet && !discharge)))
+                        && (!common ||
+                            (!orionStatus.chargeContactorOverriden && hasChargeBeenSet && !charge) ||
```

Comment:
Can we extract this to a variable? Something like
`uint8_t chargeIsOpenWithoutOrionOverride = !orionStatus.chargeContactorOverriden && hasChargeBeenSet && !charge`

Same thing with the other ones.

Reply:
I think we have another issue that's going to address compressing conditions into single variables